src/Routes/RoutesCommon.tsx

```diff
@@ -20,6 +20,9 @@ import { getConfig } from '@/config'
 import { assetIdPaths } from '@/hooks/useRouteAssetId/useRouteAssetId'
 import { FoxPage } from '@/pages/Fox/FoxPage'
 import { RFOX } from '@/pages/RFOX/RFOX'
+import { ClaimTab } from '@/pages/Trade/tabs/ClaimTab'
+import { LimitTab } from '@/pages/Trade/tabs/LimitTab'
+import { TradeTab } from '@/pages/Trade/tabs/TradeTab'
```

Comment:
Now separated into their own components for routing purposes - this is really how it should be anyway as all 3 are root-level routes (almost pages at this point)